src/pyclblast/README.md

```diff
@@ -53,6 +53,6 @@ How to release a new version on PyPi
 
 Following [the guide](https://packaging.python.org/tutorials/packaging-projects/), in essence doing (after changing the version number in `setup.py`):
 
-    python3 setup.py sdist bdist_wheel
+    python3 -m build
```

Comment:
This did not work for me, while the old command does still work:
```
:~/CLBlast/src/pyclblast$ python3 -m build
python3: No module named build.__main__; 'build' is a package and cannot be directly executed
```
And if I remove that folder then I get `No module named build`. This is with the relatively old Python 3.7.

Reply:
Hi,

No worries for the time taken, it is a festive period after all.
I have updated the changelog file.

Regarding the error, it seems to me that you have not installed the [build](https://pypi.org/project/build/) python package. Running  `python3 -m pip install --upgrade build`  (or install it through your linux distro package manager) should make it work.
Keep in mind that the `python -m build` method for packaging, is the one currently described in the guide in the `README.md` [link](https://packaging.python.org/tutorials/packaging-projects/).
Let me know if the error persists after you install the `build` package.

One more unrelated thing, it would be good to add 'hints' (if you have any suggestion) for cmake to find the CLBlast library.

Reply:
I added some hints for CLBlast in the `CMakeLists.txt` file and updated the `README.md` to inform the user about them.